### DISCUSSION TITLE
LubeLogger

### DISCUSSION THREAD

**Discussion by vrytired** [selfhst/icons#21] [New Icon Request]
Official Website: https://lubelogger.com/
Repo: https://github.com/hargata/lubelog

**Comment by shollyethan** [comment]
Struggled a bit to find a high quality version of the LubeLogger logo when I initially added it to the directory. I'll reach out to the dev via email to see if he has something he'd be willing to share.

**Comment by wiltified** [comment]
Fingers crossed on this one.  I have Photoshop / Illustrator if you need any help.

**Comment by hargata** [comment]
Idk how hi-res the icon need to be, but I have them here in a variety of sizes and transparent background:

https://docs.lubelogger.com/Misc/Branding

**Reply by shollyethan** [reply]
I can make those work - added!
